src/UCTSearch.cpp

```diff
@@ -218,6 +218,78 @@ bool UCTSearch::is_running() const {
     return m_run && m_nodes < MAX_TREE_SIZE;
 }
 
+int UCTSearch::est_playouts_left() const {
+    auto elapsed_millis = now() - m_start_time;
+    auto playouts = m_playouts.load();
+    if (m_target_time < 0) {
+        // No time control, use playouts or visits.
+        const auto playouts_left = std::min(m_maxplayouts - playouts,
+                                            m_maxvisits - m_root->get_visits());
+        return playouts_left;
+    } else if (elapsed_millis < 1000 || playouts < 100) {
+        // Wait for at least 1 second and 100 playouts
```

Comment:
"Wait for at least 1 second or 100 playouts" :)

Reply:
I think this is correct (and matches LZGo). Possibly slightly confusing because it's the `else` branch that's "at least 1 second and 100 playouts." (If so, perhaps swap them for readability?)

Reply:
I clarified the comment.